barman/config.py

```diff
@@ -70,6 +70,10 @@
 
 CREATE_SLOT_VALUES = ["manual", "auto"]
 
+# Config values relating to pg_basebackup compression
+BASEBACKUP_COMPRESSIONS = ["gzip", "lz4", "zstd"]
```

Comment:
We have to define these here because nearly every module imports config and we'd end up with a circular dependency.